### DISCUSSION TITLE
Make APKMirror an Updatable Source

### DISCUSSION THREAD

**Discussion by Hamster45105** [ImranR98/Obtainium#1711] [Ideas]
I recently saw an app that allows you to download and update apps through APKMirror. 

I don't know much about Android app developing, but I was wondering if the code in that app could be adapted for Obtainium to make APKMirror an updatable source.

**Comment by DwainZwerg** [comment]
The problem is not that Obtainium could not pull updates from APKmirror (Obtainium could already do that when there were only three sources GitHub, GitLab and APKMirror). However, the operators @archon810 do not want free clients (there is the paid APKMirror client) to download apps without advertising, the profits of which go to APKMirror (see [this comment from the operators](https://github.com/ImranR98/Obtainium/issues/44#issuecomment-1279556944)). Scraping was subsequently made more difficult for Obtainium. As a compromise, the operators and @ImranR98 agreed to make apkmirror track-only. ImranR98 then asked again in [another issue](https://github.com/ImranR98/Obtainium/issues/1387#issuecomment-1937847686) whether the download could be reactivated as with some other tools (with very low speed via an API only for Obtainium or by message that Obtainium wants to download via User-agent), but has not yet received any response.
